src/coreclr/tools/aot/crossgen2/crossgen2.props

```diff
@@ -41,12 +41,19 @@
     <Compile Include="..\..\Common\InstructionSetHelpers.cs" Link="InstructionSetHelpers.cs" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <_LibPrefix Condition="'$(_IsCrossHostPublish)' == 'true'">$(LibPrefix)</_LibPrefix>
+    <_LibPrefix Condition="'$(_IsCrossHostPublish)' != 'true'">$(HostLibPrefix)</_LibPrefix>
+    <_LibSuffix Condition="'$(_IsCrossHostPublish)' == 'true'">$(LibSuffix)</_LibSuffix>
+    <_LibSuffix Condition="'$(_IsCrossHostPublish)' != 'true'">$(HostLibSuffix)</_LibSuffix>
```

Comment:
@steveisok do we still need these changes? 
The last commit removed setting `_IsCrossHostPublish`

Reply:
Fixed. `<_LibPrefix|Suffix>` is still needed because there would be host/target confusion trying to cross build for linux on a mac.